src/logic/events/TrackpadEvents.swift

```diff
@@ -0,0 +1,113 @@
+import M5MultitouchSupport
+
+//TODO: patch M5MultitouchSupport pod to make it not to crash after sleep. See https://github.com/mhuusko5/M5MultitouchSupport/issues/1
```

Comment:
I'm a bit worried to rely on a third-party library that hasn't been updated in 8 years, and that has a crash on wake that has only been found this year.

Is there no way to roll our own code for multi-fingers gesture? Is it too rough when using the AppKit APIs?

Reply:
**Update:** i just searched online for this, and there is no information out there. I understand better why you went with this library. Classic Apple move to not properly support such a classic interaction decades after it's been added to macOS.

I've updated the issue on the `M5MultitouchSupport` repo. Maybe we could add something like this to fix the issue.

Reply:
I don't want to add a 3-rd party library either but it's just a PoC.
We could patch M5MultitouchSupport as I did it for Touch-Tab https://github.com/ris58h/Touch-Tab/blob/master/patches/M5MultitouchSupport%2B1.1.0.diff (it works fine) or use any other library.
Seems like https://github.com/Kyome22/OpenMultitouchSupport is a good candidate.

P.S.:
Code in both libraries is nearly identical ([M5MultitouchSupport](https://github.com/mhuusko5/M5MultitouchSupport/blob/master/M5MultitouchSupport/M5MultitouchManager.m#L214) vs [OpenMultitouchSupport](https://github.com/Kyome22/OpenMultitouchSupport/blob/master/OpenMultitouchSupport/OpenMTManager.m#L121)) but OpenMultitouchSupport checks if [MTDeviceIsRunning](https://github.com/Kyome22/OpenMultitouchSupport/blob/master/OpenMultitouchSupport/OpenMTManager.m#L122) before releasing the device.

Reply:
It turned out that OpenMultitouchSupport has the same issue https://github.com/Kyome22/OpenMultitouchSupport/issues/2 It seems like we have to patch anyway.

Should I commit changes in `Pods` dir after a new pod is installed?